### PR TITLE
Code refactor

### PR DESCRIPTION
**Pull request by CuriousLearner**
This PR aims at cleaning the code and create a proper directory structure for maintenance.

### CODE REVIEW

**Comment by CuriousLearner** [comment]
@RamolaWeb Would you like to review this PR?

File: g4g.py [hunk 1]
```diff
@@ -29,48 +38,55 @@ def display_menu():
...
-    allArticles += '<h1 style="text-align:center;font-size:40px">' + categoryUrl.title() + ' Archive</h1><hr>'
-    allArticles += '<h1 style="padding-left:5%;font-size:200%;"> Index</h1><Br/>'
+    all_articles = (
+        '<!DOCTYPE html>'
+        '<html><head>'
+        '<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />'
+        '<link rel="stylesheet" href="style.min.css" type="text/css" media="all" />'
+        '<script src="https://cdn.rawgit.com/google/code-prettify/master/loader/run_prettify.js"></script>'
+        '</head><body>'
+    )
```
Comment (RamolaWeb) [inline g4g.py]: The Closing Brackets Does not matching any visual indentation.

File: g4g.py [hunk 2]
```diff
@@ -1,19 +1,28 @@
...
-                      5: 'fundamentals-of-algorithms',
-                      6: 'data-structures'}
+CHOICE_TO_CATEGORY_MAPPING = {
+    1: 'c',
+    2: 'c-plus-plus',
+    3: 'java',
+    4: 'python',
+    5: 'fundamentals-of-algorithms',
+    6: 'data-structures'
+}
```
Comment (RamolaWeb) [inline g4g.py]: The Closing Brackets does not match any Visual Indentation

Comment (CuriousLearner) [inline g4g.py]: Can you please explain this?

Comment (RamolaWeb) [inline g4g.py]: @CuriousLearner  I have use a code Linter to lint the code and at this line it was saying that The Closing Brackets does not match any Visual Indentation. That means the closing brackets of the code are not indented properly to match the starting brackets

Comment (CuriousLearner) [inline g4g.py]: I guess both are valid syntax like

```
a = {
    'some_key': 'some_value'
}
```

and 

```
a = {
    'some_key': 'some_value'
    }
```

Comment (RamolaWeb) [inline g4g.py]: @CuriousLearner  Ok Then I think the code is clean now

**Comment by CuriousLearner** [comment]
@RamolaWeb Can you try running the script again and verify here if everything looks good?

Then we can merge this.

**Comment by CuriousLearner** [comment]
Hi @RamolaWeb Please let me know here once you verify these changes, and I'll merge them in!

Thanks!

**Comment by RamolaWeb** [comment]
@CuriousLearner  Everything look good , I have Tested the Code till here and everything look correct . The linking in the pdf is not working but it may be the problem of the library. I think we can merge here to add new feature to it.

**Comment by CuriousLearner** [comment]
@RamolaWeb Yes, that's an issue as we've seen on the repo of `wkhtmltopdf`.
I'll merge this in, so that next tasks could be worked upon.

Thanks for the help!

**Pull request merged**